src/org/thoughtcrime/securesms/ApplicationPreferencesActivity.java

```diff
@@ -126,6 +126,7 @@ public boolean onSupportNavigateUp() {
   @Override
   public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
     if (key.equals(Prefs.THEME_PREF)) {
+      DynamicTheme.setDefaultDayNightMode(this);
```

Comment:
was that always missing/wrong?

i am wondering what that has to do with contact requests :)

Reply:
a modern approach was used for the dark theme version of the buttons using `drawable-night` it is nicer than the current approach in other parts, but I hope that is compatible with old Android versions?

Reply:
i did not tested it on android 4, however, i assume it is fine as these drawable-_someVersionOrConfig_ are usually just ignored by older androids.